compiler/rustc_mir_transform/src/inline.rs

```diff
@@ -424,6 +415,10 @@ impl<'tcx> Inliner<'tcx> {
         if callee_body.basic_blocks.len() <= 3 {
             threshold += threshold / 4;
         }
+
+        if tcx.sess.mir_opt_level() == 1 || tcx.sess.opts.incremental.is_some() {
+            threshold = 2 * INSTR_COST;
```

Comment:
This is twice INSTR_COST right now because a return terminator gets INSTR_COST, so if the threshold were just INSTR_COST that would only inline functions that are just a terminator.

Reply:
Would be good to have that as a comment in the code